src/applications/disability-benefits/view-payments/components/responsive-table/ResponsiveTable.scss

```diff
@@ -32,13 +32,13 @@ table.responsive td {
 
   table.responsive tr {
     display: block;
-    margin-bottom: 0.625em;
+    border-bottom: 1px solid black;
```

Comment:
I _think_ the borders for this table are meant to be #757575, based on the [mockups](https://xd.adobe.com/view/5f609803-b045-428a-510e-1754e404af3e-2951/screen/6a2de0f2-3a3b-4da0-b252-471e5cdcb7e9/specs/). Would you mind just double checking with Jim?

Reply:
good catch! updated